river/Output.zig

```diff
@@ -60,6 +60,20 @@ pub const PendingState = struct {
     ///
     /// This includes both floating/fullscreen views and those arranged in the layout.
     wm_stack: wl.list.Head(View, .pending_wm_stack_link),
+    ///
+    /// Index of the currently focused view in the wm_stack list.
+    pub fn focused_index(self: *PendingState) ?u32 {
```

Comment:
Though I too prefer snake_case, let's keep this camelCase for consistency for now
```suggestion
    pub fn focusedIndex(self: *PendingState) ?u32 {
```

Maybe I'll get rebellious and reformat all of river to snake_case functions at some point despite https://github.com/ziglang/zig/issues/1097 not being accepted yet.